build.proj

```diff
@@ -50,9 +51,10 @@
       <InputOSGroup>$(InputOSGroup)</InputOSGroup>
     </Project>
     <Project Include="src\post.builds">
```

Comment:
><Project Include="src\post.builds"> [](start = 4, length = 35)

I know tests.builds will now be only invoked when calling build-tests.cmd, but what is the plan for the rest, like post.builds, sign.builds and packages.builds?